src/traffic-collector/azext_traffic_collector/azext_metadata.json

```
@@ -0,0 +1,4 @@
+{
+    "azext.isExperimental": true,
```

Do you want to release this extension in experimental, preview or GA?

---

@kairu-ms Our service is currently in public preview and will be in GA next month. Given this, what do you suggest on where should we release our extension?

---

I recommend to marks it as preview first, as it's the first version of extension. You can GA it in the next version.
```suggestion
    "azext.isPreview": true,
```

---

If you want to release a GA version, simplify remove line 2.

---

You can set it as `"azext.isPreview": true` instead of `"azext.isExperimental": true`.

---

@kairu-ms Currently the traffic-collector extension is marked as "preview". Our service is planned to go GA starting November. Should we be changing this after we go to GA or can we change the CLI scope before we go GA?